client/mm/libxc/bitget.go

```
@@ -0,0 +1,3133 @@
+				continue
+			}
+
+			// Ensure lot size is at least 1
+			if lotSize == 0 {
+				lotSize = 1
+			}
+
+			minWithdraw[assetID] = &bitgetWithdrawInfo{
+				minimum: minimum, // max(deposit_min, withdrawal_min) to satisfy both
+				lotSize: lotSize,
+			}
+		}
+	}
+
+	bg.log.Debugf("Stored deposit/withdrawal info for %d assets", len(minWithdraw))
+	bg.tokenIDs.Store(tokenIDs)
+	bg.minWithdraw.Store(minWithdraw)
+}
+
+// getMarkets retrieves and processes market/symbol information
+func (bg *bitget) getMarkets(ctx context.Context) (map[string]*bgtypes.Market, error) {
+	var resp bgtypes.APIResponse
+	err := bg.getAPI(ctx, "/api/v2/spot/public/symbols", nil, false, &resp)
+	if err != nil {
+		return nil, err
+	}
+
+	if resp.Code != "00000" {
+		return nil, fmt.Errorf("bitget API error: %s - %s", resp.Code, resp.Msg)
+	}
+
+	var symbols []*bgtypes.SymbolInfo
+	if err := json.Unmarshal(resp.Data, &symbols); err != nil {
+		return nil, fmt.Errorf("error unmarshaling symbols: %w", err)
+	}
+
+	marketsMap := make(map[string]*bgtypes.Market, len(symbols))
+	tokenIDs := bg.tokenIDs.Load().(map[string][]uint32)
+
+	skippedNotLive := 0
+	skippedNoMatch := 0
+
+	for _, sym := range symbols {
+		// Bitget uses "online" for active markets, not "live"
+		if sym.Status != "online" {
+			skippedNotLive++
+			continue
+		}
+
+		// Check if we support both assets
+		dexMarkets := bitgetMarketToDexMarkets(sym.BaseCoin, sym.QuoteCoin, tokenIDs)
+		if len(dexMarkets) == 0 {
+			skippedNoMatch++
+			continue
+		}
+
+		dexMkt := dexMarkets[0]
+		bui, _ := asset.UnitInfo(dexMkt.BaseID)
+		qui, _ := asset.UnitInfo(dexMkt.QuoteID)
+
+		minTradeAmount, _ := strconv.ParseFloat(sym.MinTradeAmount, 64)
+		maxTradeAmount, _ := strconv.ParseFloat(sym.MaxTradeAmount, 64)
+		minTradeUSDT, _ := strconv.ParseFloat(sym.MinTradeUSDT, 64)
+		pricePrecision, _ := strconv.Atoi(sym.PricePrecision)
+		quantityPrecision, _ := strconv.Atoi(sym.QuantityPrecision)
```

Would rather not ignore any errors. Please log.